src/BlazorWebService/BlazorWebService.Client/Pages/Weather.razor

```diff
@@ -0,0 +1,48 @@
+﻿@page "/weather"
+@rendermode @(new InteractiveWebAssemblyRenderMode(false))

```

Comment:
I missed this earlier. I was looking for `<Routes @rendermode="new InteractiveWebAssemblyRenderMode(prerender: false)" />`, but didn't see it at first. Wherever we put it, we should write out `prerender: false` because otherwise it's not clear what `false` means.

I now see that this is a modification of the auto per-page template. Do we want to change the auto template? I thought we were only going to change the WebAssembly template.

Also, if we think that disabling prerendering for WebAssembly is a common enough scenario that we want it in our project templates, should we add a static property to `RenderMode` like `NonPrerenderedInteractiveWebAssembly`? That's a lot less characters than `@(InteractiveWebAssemblyRenderMode(prerender: false))`.

Reply:
I think we want to have the same Blazor WebAssembly client project for both auto and WebAssemblyInteractive cases, don't we?

Reply:
Disabling prerendering when using WebAssembly helps because you can then assume your components will only run on the client. It doesn't help to disable prerendering when using Auto because with your components will run on both the client and server.

Reply:
Got it. I'll make sure this applies to only the WebAssembly interactive mode only.

Thanks guys!

Reply:
Updated to use the WebAssembly interactivity as the base project.

Reply:
> should we add a static property to RenderMode like NonPrerenderedInteractiveWebAssembly?
I would like to keep the change scoped to the template only, and avoid adding other changes for now. If we want to do that, we should handle that as part of a separate change.